src/index.js

```diff
@@ -3,12 +3,17 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import {BrowserRouter ,Routes, Route}  from 'react-router-dom'
```

Comment:
comma/spacing issue - will be fixed by linter

Reply:
done